src/write.jl

```diff
@@ -278,9 +293,23 @@ function Base.close(writer::Writer)
     nothing
 end
 
-function write(io::IO, tbl; kwargs...)
+function write(io::IO, tbl; chunksize::Union{Nothing,Integer}=64000, kwargs...)
+    # rowaccces is a necessary pre-requisite for row-iteration (not sufficient though)
+    if !isnothing(chunksize) && Tables.istable(tbl) && Tables.rowaccess(tbl)
+        @assert chunksize >= 0 "chunksize must be >= 0"
+        if hasmethod(Iterators.partition,(typeof(tbl),))
+            tbl_source = Iterators.partition(tbl, chunksize)
```

Comment:
Can we use `Iterators.partition` from Base rather than `DataFrames` to prevent adding one more dependency?

https://docs.julialang.org/en/v1/base/iterators/#Base.Iterators.partition